client/services/apiService.ts

```diff
@@ -37,16 +37,26 @@ export const getSelectedProfile = async (profileId: number): Promise<ProfileType
   }
 };
 
-// export const getAllStoriesByProfile = async (profileId: number): Promise<StoryType[]> => {};
+export const getAllStoriesByProfile = async (profileId?: number): Promise<StoryType[]> => {
```

Comment:
`profileId?:` what does the question mark do here? this should not be an optional argument